packages/contracts-bedrock/scripts/deploy/ChainAssertions.sol

```diff
@@ -134,6 +136,32 @@ library ChainAssertions {
         }
     }
 
+    /// @notice Asserts that the SystemConfigInterop is setup correctly
+    function checkSystemConfigInterop(
+        Types.ContractSet memory _contracts,
+        DeployConfig _cfg,
+        bool _isProxy
+    )
+        internal
+        view
+    {
+        ISystemConfigInterop config = ISystemConfigInterop(_contracts.SystemConfig);
+        console.log(
+            "Running chain assertions on the SystemConfigInterop %s at %s",
+            _isProxy ? "proxy" : "implementation",
+            address(config)
+        );
+
+        checkSystemConfig(_contracts, _cfg, _isProxy);
+        if (_isProxy) {
+            // TODO: this is not being set in the deployment, nor is a config value.
+            // Update this when it has an entry in hardhat.json
+            require(config.dependencyManager() == address(0), "CHECK-SCFG-10");
```

Comment:
In OPCMInterop we do set this, so I assume this check would start to fail once we integrate OPCM?

https://github.com/ethereum-optimism/optimism/blob/37dd9b3374c562c9e2b9981f4ed596a7f0d92fe0/packages/contracts-bedrock/src/L1/OPContractsManagerInterop.sol#L40

Reply:
Yes. Although I'll need to just hardcode some dummy address to pass to the OPCM. What I'll do then is also pass that as and `expectedDependencyManager` arg to this function.